hack/install-operator-sdk.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+
+OPERATOR_SDK_VERSION="v1.24.0"
+OPERATOR_SDK_BASE_URL="https://github.com/operator-framework/operator-sdk/releases/download/"
+
+OPERATOR_SDK_INSTALLED_VER=$("${SCRIPT_DIR}/../bin/operator-sdk" version | cut -d" " -f3 | tr -d "," | tr -d '"')
+
+if [ -z "$OPERATOR_SDK_INSTALLED_VER" ] || [ "$OPERATOR_SDK_VERSION" != "$OPERATOR_SDK_INSTALLED_VER" ]; then
+  if [ -n "$OPERATOR_SDK_INSTALLED_VER" ]; then
+    echo "Incorrect version ($OPERATOR_SDK_INSTALLED_VER) of operator-sdk found, expecting $OPERATOR_SDK_VERSION"
+    rm -f "${SCRIPT_DIR}/../bin/operator-sdk"
+  fi
+
+  echo "Installing operator-sdk (version: $OPERATOR_SDK_VERSION) into bin"
+  mkdir -p "${SCRIPT_DIR}/../bin"
+  OS=$(go env GOOS)
+  ARCH=$(go env GOARCH)
+  curl -sSLo "${SCRIPT_DIR}/../bin/operator-sdk" "${OPERATOR_SDK_BASE_URL}/${OPERATOR_SDK_VERSION}/operator-sdk_${OS}_${ARCH}"
```

Comment:
curl -sSLo path works, but it will be more clear to use long options names and separate the option that need argument from the rest:

```
curl --silent --show-error --location --output "$target_path" "$source_url"
```

Reply:
Done